test/.package.json

```diff
@@ -10,7 +10,12 @@
   "author": "Tim von Oldenburg <tim.vonoldenburg@skyscanner.net>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "eslint-config-skyscanner": "file:../"
+    "babel-eslint": "^8.0.1",
```

Comment:
I need to add all of `eslint-config-skyscanner`'s peer dependencies in order to test this.